loom/vendor/stb/stb_truetype.h

```diff
@@ -630,7 +630,7 @@ enum { // languageID for STBTT_PLATFORM_ID_MAC
 static int stbtt__isfont(const stbtt_uint8 *font)
 {
    // check the version number
-   if (stbtt_tag(font, "1"))   return 1; // TrueType 1
+   if (font[0] == '1')            return 1; // TrueType 1
```

Comment:
If you look a bit above this line, you can see that in the definition:
```
#define stbtt_tag4(p,c0,c1,c2,c3) ((p)[0] == (c0) && (p)[1] == (c1) && (p)[2] == (c2) && (p)[3] == (c3))
#define stbtt_tag(p,str)           stbtt_tag4(p,str[0],str[1],str[2],str[3])
```
it directly accesses [0], [1], [2], [3], which seems like a very bad thing if you pass "1" to it. So I just directly checked the first byte, which is what I assume was meant in the first place.